docs/sources/tempo/configuration/grafana-alloy/tail-sampling/policies-strategies.md

```diff
@@ -0,0 +1,407 @@
+
+Refer to the <code>[latency](https://grafana.com/docs/alloy/latest/reference/components/otelcol/otelcol.processor.tail_sampling/#latency)</code> documentation for more information.
+
+```alloy
+policy {
+    // The example below is for the `latency` sampling policy
+    name = "example_latency"
+    type = "latency"
+
+    latency {
+      // The minimum duration for a trace to be sampled
+      threshold_ms = 3000
+      // The maximum duration for a trace to be sampled
+      max_duration_ms = 10000
+    }
+  }
+```
+
+### Numeric attribute
+
+The `numeric_attribute` policy lets you sample based on a number of attributes. In this example below, spans are sampled where `http.response_content_length` is between 10,000 and 500,000. The sample span below would be captured because the attribute value falls within the specified range.
+
+Refer to the [`numeric_attribute`](https://grafana.com/docs/alloy/latest/reference/components/otelcol/otelcol.processor.tail_sampling/#numeric_attribute) documentation for more information.
+
+```alloy
+policy {
+    // The example below is for the `numeric_attribute` sampling policy
+    name = "example_numeric_attribute"
+    type = "numeric_attribute"
+
+    numeric_attribute {
+      // The span or resource attribute to be considered.
+      key = "http.response_content_length"
+      // The minimum value for the attribute to be sampled.
+      min_value = 10000
+      // The maximum value for the attribute to be sampled.
+      max_value = 500000
+    }
+  }
+```
+
+### OTTL condition
+
+This policy type lets you configure policies based around OTTL. The various language semantics let you write flexible and detailed sampling policies that can evaluate any trace span conditions to determine whether or not to sample a trace.
+If OTTL conditions are particularly complex, this flexibility can potentially create resource overhead.
+
+Refer to the [`ottl_condition`](https://grafana.com/docs/alloy/latest/reference/components/otelcol/otelcol.processor.tail_sampling/#ottl_condition) documentation for more information.
+
+```alloy
+policy {
+    // The example below is for the `ottl_condition` sampling policy
+    name = "example_ottl_condition"
+    type = "ottl_condition"
+
+    ottl_condition {
+        condition = "resource.attribute['service.name'] == 'checkout_service'"
+    }
+  }
+```
+
+### Probabilistic
+
+Probabilistic sampling lets you determine a ratio of traces that should be sampled. This is within a range of 0 to 100%, and selects the required percentage of traces from those it receives based on the number of traces per second configured. This can be combined with varying hashing salt values, should multiple collectors be carrying out sampling simultaneously (for example in a Load Balanced sampling hierarchy).
```

Comment:
```suggestion
Probabilistic sampling lets you determine a ratio of traces that should be sampled.
This is within a range of `0` to `100%`, and selects the required percentage of traces from those it receives based on the number of traces per second configured.
This can be combined with varying hashing salt values, should multiple collectors be carrying out sampling simultaneously, for example, in a load balanced sampling hierarchy).
```